.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "arrow-parens": "off"
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/",
+      "src/",
+      "webpack-config.js"
```

Comment:
Please do not make any changes to the config files. 
![image](https://user-images.githubusercontent.com/38625956/88806827-be0bc080-d1d2-11ea-8f3f-ebd20c333e1b.png)
For resolving them please read the error messages carefully the solution is also in there.